<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>506</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视频播放器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog  { background-color: #f2f1f5; border-radius: 22px; }
* { color: #3d4150; }
QPushButton { border-style: none;border: 1px solid #DCDFE6;border-radius: 6px;padding: 5px 10px;font-size: 13px; }
QPushButton:hover { color: #b2b7cb; font-weight: bold; background-color: #1296db; }
QScrollBar:vertical { background:#f5f6f8; padding:0px; border-radius: 3px; width:6px; }
QScrollBar::handle:vertical { background:#C0C4CC; border-radius: 3px; }
QScrollBar::handle:vertical:hover { background:#909399; }
QScrollBar::handle:vertical:pressed { background:#909399; }
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical, QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical { background:none; }
QCheckBox, QRadioButton { font-size: 13px; }
QComboBox, QLineEdit, QTextEdit, QTextBrowser, QTimeEdit, QDateEdit, QDateTimeEdit, QSpinBox,
QDoubleSpinBox { background-color: #ffffff; border: 1px solid #DCDFE6; border-radius: 6px; padding: 5px 10px; font-size: 13px; }
QTextEdit, QTextBrowser { padding-right: 3px; }
QLineEdit:hover, QComboBox:hover, QTimeEdit:hover, QDateEdit:hover, QDateTimeEdit:hover, QSpinBox:hover, QDoubleSpinBox:hover { border: 1px solid #409EFF; }
QComboBo::down-arrow { width:10px; height:10px; right:2px; image:url(:/icons/down.png); }
QComboBo::drop-down { subcontrol-origin:padding; subcontrol-position:top right; width:15px; border-left-width:0px; border-left-style:solid; border-top-right-radius:3px; border-bottom-right-radius:3px; border-left-color:#242424; }
QComboBox QAbstractItemView::item{ min-height:20px; min-width:10px; }
QComboBo::drop-down:on{ top: 1px; }
QTimeEdit::up-button, QDateEdit::up-button, QDateTimeEdit::up-button, QSpinBo::up-button, QDoubleSpinBo::up-button { width:10px; height:10px; image:url(:/icons/up.png); padding: 3px 8px 0px 0px; }
QTimeEdit::down-button, QDateEdit::down-button, QDateTimeEdit::down-button, QSpinBo::down-button, QDoubleSpinBo::down-button { width:10px; height:10px; image:url(:/icons/down.png); padding: 0px 8px 3px 0px; }
QTimeEdit::up-button:pressed, QDateEdit::up-button:pressed, QDateTimeEdit::up-button:pressed, QSpinBo::up-button:pressed, QDoubleSpinBo::up-button:pressed { top: -2px; }
QTimeEdit::down-button:pressed, QDateEdit::down-button:pressed, QDateTimeEdit::down-button:pressed, QSpinBo::down-button:pressed, QDoubleSpinBo::down-button:pressed { bottom: -2px; }
QTabWidget::pane{ border:1px solid #DCDFE6; border-radius: 6px; border-top-left-radius: 0px; background-color: #ffffff; selection-background-color:#ffffff; selection-color:#DCDFE6; alternate-background-color:#f2f1f5; gridline-color:#3d4150; }
QTabBar::tab { border:1px solid #DCDFE6; border-top-left-radius: 6px; border-top-right-radius: 6px; font-size: 13px; background-color: #f2f1f5; padding: 2px 6px; }
QTabWidget::pane:top { top:-1px; }
QTabWidget::pane:bottom { bottom:-1px; }
QTabWidget::pane:left { right:-1px; }
QTabWidget::pane:right { left:-1px; }
QTabBar::tab:selected { background-color: #ffffff; border-bottom: none; }
QGroupBox { border: 1px solid #DCDFE6; border-radius: 6px; font-size: 13px; margin-top: 7px; }
QGroupBox::title { subcontrol-origin: margin; subcontrol-position: top center; padding: 0px 5px 3px; }

#widget_menu { background-color: #f8f8f8; border-radius: 16px; }
#btn_playVideo, #btn_playMusic, #btn_ScreenRecording, #toolBtn_screenShot { background-color: #ffffff; border-radius: 5px; padding: 12px 0 0; }
#btn_playVideo:hover, #btn_playMusic:hover, #btn_ScreenRecording:hover, #toolBtn_screenShot:hover,hover { background-color: #f8f8f8; border: 1px solid #f0f0f0; }
#lable_title { padding: 0 10px; }
#btn_language, #btn_meeting, #btn_normal, #btn_notion { padding: 8px 12px; color: #d1d8e7; background-color: #b8c3ce;  border: none; }
#btn_language:hover, #btn_meeting:hover, #btn_normal:hover, #btn_notion:hover { background-color: #ffffff; border-radius: 15px; color: #b8c3ce; }
#btn_personal_homePage, #btn_quit, #btn_scaledToSmall, #btn_littleshow {  border: none; }
#btn_personal_homePage:hover, #btn_quit:hover, #btn_scaledToSmall:hover { background-color: #eeeff4; border-radius: 10px; }
#stackedWidget { background-color: #f8f8f8; border-radius: 16px; }
#page_memberShip, #page_normal, #page_notion, #page_words { color: #C0C4CC; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <item>
      <widget class="QVideoWidget" name="widgetvideo" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_start">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_stop">
        <property name="text">
         <string>回到开始</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_openFile">
        <property name="text">
         <string>选取文件</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_speedBack">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../MyMusic/resMusic.qrc">
          <normaloff>:/images/images/s.png</normaloff>:/images/images/s.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="m_startButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MyMusic/resMusic.qrc">
          <normaloff>:/images/images/bf.png</normaloff>:/images/images/bf.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="m_stopButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../MyMusic/resMusic.qrc">
          <normaloff>:/images/images/zt.png</normaloff>:/images/images/zt.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="btn_speedFoward">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../MyMusic/resMusic.qrc">
          <normaloff>:/images/images/x.png</normaloff>:/images/images/x.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QSlider" name="horizontalSlider">
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {  
    border: 1px solid #999999;  
    background: #ffffff;  
    height: 8px;  
    margin: 0px;  
    round-corners: both;  
}  
  
QSlider::handle:horizontal {  
    background: #66cc99;  
    border: 1px solid #555555;  
    width: 8px;  
    margin: -7px 0; /* margin top/bottom = -(handle height/2) */  
    border-radius: 4px;  
}  
  
QSlider::handle:horizontal:hover {  
    background: #77ddaa;  
}  
  
QSlider::sub-page:horizontal {  
    background: #99cc99;  
    border: 1px solid #777777;  
    height: 8px;  
}  
  
QSlider::add-page:horizontal {  
    background: none;  
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QComboBox" name="combo_speed">
        <item>
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.75</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="btn_volumn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../MyMusic/resMusic.qrc">
          <normaloff>:/images/images/yl.png</normaloff>:/images/images/yl.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="btn_fullScreen">
        <property name="text">
         <string>全屏</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="label_duration">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MyMusic/resMusic.qrc"/>
 </resources>
 <connections/>
</ui>
